genesis/make_genesis_file.go

```diff
@@ -28,6 +28,25 @@ func GenerateKnown(chainID, accountsPathCSV, validatorsPathCSV string) (string,
 }
 
 //------------------------------------------------------------------------------------
+// interface functions that are consumed by monax tooling
+
+func GenerateGenesisFileBytes(chainName string, genesisAccounts []*GenesisAccount,
```

Comment:
Similar to other comment, if it's convenient to pass `genesisAccounts []GenesisAccount` then we could probably get away without a clone... (though we do have pointers from `GenesisAccount` to `AccountPermissions`, but I don't think we accidentally mutate here)